tests/test_distortion.py

```diff
@@ -26,6 +27,12 @@ def tf_ms_ssim(x, y):
     return torch.tensor(tf.image.ssim_multiscale(x, y, max_val=1).numpy())
 
 
+def rand_im(shape, seed=12345):
```

Comment:
Wouldn't it be easier to just use `torch.manual_seed(seed)` at the top of each test and just keep the original code, therefore you wouldn't have to pass a different seed every time you created a new tensor?

Alternatively, we could add a `setup_function` in the file that automatically sets the seeds before each tests, as per [the pytest docs](https://docs.pytest.org/en/6.2.x/xunit_setup.html).

Reply:
I don't think so. That will change the global seed. The seed should be local to each test.